beta-src/src/components/ui/WDSaveReadyButtons.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import { useState } from "react";
+import { Stack } from "@mui/material";
+import WDButton from "./WDButton";
+
+const WDSaveReadyButtons: React.FC = function () {
+  const [saveSelect, setSaveSelect] = useState(false);
+  const [readySelect, setReadySelect] = useState(false);
```

Comment:
i am pretty sure we're going to be moving this state up the parent and these are going to be props.